src/ui/movable_rect.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <types.hpp>
+
+namespace nebula {
+  /**
+   * Movable rectangle ( in short MR ) is movable visual copy of MR Button
+   *created
+   *after interaction with it. This structure holds information about objects
+   *current location in the application window and pointer to render function
+   *inherited from MR Button.
+   **/
+  using Movable_Rect = struct Movable_Rect {
+    /** window_width - window width. Important for correct position
+     *                       calculation during movement
+     */
+    static f32 window_height;
```

Comment:
is it the height of the OS window? should not be static I believe. if you need it, source it from outside via a parameter.
if it is the height of the rect, then it should be per instance (not static).

Reply:
Passed as param in new patch set